pkg/storage/utils/volumes.go

```diff
@@ -0,0 +1,121 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright The KubeVirt Authors.
+ *
+ */
+
+package utils
+
+import (
+	"context"
+	"fmt"
+
+	k8sv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "kubevirt.io/api/core/v1"
+	snapshotv1 "kubevirt.io/api/snapshot/v1beta1"
+	"kubevirt.io/client-go/kubecli"
+
+	backendstorage "kubevirt.io/kubevirt/pkg/storage/backend-storage"
+)
+
+const (
+	FLAG_EXCLUDE_NONE    = 0
+	FLAG_EXCLUDE_BACKEND = 1 << iota
+	// TODO: Add other flags as needed (KERNEL_BOOT?)
+)
```

Comment:
I perfer also a positive meaning here and a proper type of the options:
```golang
type VolumeOption int

const ( 
     WithBackandVolume VolumeOption = 0
     WithKernelBootVolume VolumeOption = iota
)
```

Reply:
Ack the first comment. Regarding the second one, I think it still doesn't make too much sense since using `WithBackendVolume` will include them all.